bin/create-wallet.rs

```diff
@@ -0,0 +1,25 @@
+use aa_bundler::models::wallet::Wallet;
+use anyhow::Result;
+use clap::Parser;
+use expanded_pathbuf::ExpandedPathBuf;
+
+#[derive(Parser)]
+#[clap(
+    name = "aa-bundler-create-wallet",
+    about = "Bundler's wallet creation for EIP-4337 Account Abstraction"
+)]
+pub struct Opt {
+    #[clap(long, default_value = "./src/res/bundler")]
```

Comment:
I think it would be better to use `$HOME/.aa-bundler` or just current dir for default_value.